dist/stat_data.py

```diff
@@ -207,6 +207,7 @@ def __init__(self, name, desc, flags=''):
     CacheStat('cache_eviction_force', 'pages evicted because they exceeded the in-memory maximum'),
     CacheStat('cache_eviction_force_delete', 'pages evicted because they had chains of deleted items'),
     CacheStat('cache_eviction_force_fail', 'failed eviction of pages that exceeded the in-memory maximum'),
+    CacheStat('cache_eviction_force_fail_time', 'time spent on failed eviction of pages that exceeded the in-memory maximum (usecs)'),
```

Comment:
It's better when the descriptions for pairs of stats have matching prefixes, so they sort together when viewed in tools.